lab_3_keywords_textrank/main.py

```diff
@@ -41,6 +41,7 @@ class TextPreprocessor:
     preprocess_text(text: str) -> tuple[str, ...]:
         Produces filtered clean lowercase tokens from raw text
     """
+
```

Comment:
revert the files of 3 lab to the master version

Reply:
done